app/frontend/styles/components/checkboxes.scss

```diff
@@ -4,3 +4,41 @@
     @extend .govuk-checkboxes__label;
   }
 }
+
+.checkboxes-with-scroll {
+  overflow-y: auto;
+  overflow-x: hidden;
+  max-height: 300px;
+  padding: 5px 15px 5px 15px;
+  border-left: 5px solid #dee0e2;
+  border-right: 5px solid #dee0e2;
+  border-bottom: 5px solid #dee0e2;
+
+  .govuk-checkboxes__item label{
+    font-weight: 600;
+  }
+}
+
+.checkboxes-with-scroll__filter {
+  position: relative;
+  background: #ffffff;
+  border-left: 5px solid #dee0e2;
```

Comment:
should use a GDS color or its nearest equivalent here

Reply:
Yeah these need to be updated, all css is just copy and pasted from the prototype at the minute